source/rules/no-ignored-subscription.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { Rule } from "eslint";
+import * as es from "estree";
+import * as ts from "typescript";
+import { couldBeType } from "tsutils-etc";
+import { getTypeCheckerAndNodeMap } from "../utils";
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      category: "RxJS",
+      description: "Disallows ignoring the subscription returned by subscribe.",
+      recommended: true
+    },
+    fixable: null,
+    messages: {
+      forbidden: "Ignoring returned subscriptions is forbidden."
+    },
+    schema: []
+  },
+  create: context => {
+    return {
+      "ExpressionStatement[expression.callee.property.name='subscribe']:has([expression.arguments.length = 0]) > CallExpression > MemberExpression": (
+        node: es.MemberExpression
+      ) => {
+        const { nodeMap, typeChecker } = getTypeCheckerAndNodeMap(context);
```

Comment:
Minor nit, but I think that calls that depend only upon the `context` should be made in the `create` implementation - rather than in the listener implementation.